rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -12697,16 +12673,14 @@ bool opcode_0424(const ScriptArguments& args) {
 }
 
 /**
-	@brief convert_metres_to_feet %2d% store_to %1d%
+	@brief %2d% = meters_to_feet %1d% 
 
 	opcode 0425
 	@arg arg1 
 	@arg arg2 
 */
 void opcode_0425(const ScriptArguments& args, const ScriptFloat arg1, ScriptFloat& arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x0425);
-	RW_UNUSED(arg1);
-	RW_UNUSED(arg2);
+	arg2 = arg1 * 3.2808398;
```

Comment:
This is actually too accurate, the behaviour is documented on https://www.gtamodding.com/wiki/0425

Reply:
Multiplication with 3 will be enough?

Reply:
More like 3.333333, or division by 0.3